impl/client/src/main/javascript/web/pentaho/visual/views/echarts/Funnel.js

```diff
@@ -88,7 +88,7 @@ define([
           name: dataTable.getCompositeFormattedValue(i, categoriesColIndexes, this.groupedLabelSeparator),
           value: dataTable.getValue(i, measureColIndex),
           tooltip: this._buildTooltip(this._buildRowTooltipHtml(dataTable, i), font),
-          visualKey: dataTable.getCompositeValue(i, categoriesColIndexes, this.groupedLabelSeparator)
+          visualKey: this._getVisualKey(dataTable.getRowCells(i, categoriesColIndexes))
```

Comment:
The visual key should be directly the result of getRowCells.

Reply:
getRowCells returns an objects' array . Since, only key is needed, isn't it better to just take that ?🤔
![image](https://github.com/pentaho/pentaho-platform-plugin-common-ui/assets/113503744/230b7b09-6fe0-466a-896c-edce421891e5)